src/Sulu/Bundle/DocumentManagerBundle/Initializer/WorkspaceInitializer.php

```diff
@@ -35,7 +35,7 @@ public function __construct(ConnectionRegistry $registry)
      */
     public function initialize(OutputInterface $output)
     {
-        foreach ($this->registry->getConnections() as $connection) {
+        foreach ($this->registry->getConnections() as $name => $connection) {
```

Comment:
Depends, this is a WIP. It seems to work without the admin connectio
here, but I am not sure why.

On Mon, Mar 14, 2016 at 10:04:00AM -0700, Daniel Rotter wrote:
>    In
>    [1]src/Sulu/Bundle/DocumentManagerBundle/Initializer/WorkspaceInitializer.php:
> 
>  > @@ -35,7 +35,7 @@ public function __construct(ConnectionRegistry $registry)
>  >       */
>  >      public function initialize(OutputInterface $output)
>  >      {
>  > -        foreach ($this->registry->getConnections() as $connection) {
>  > +        foreach ($this->registry->getConnections() as $name => $connection) {
> 
>    The name isn't required here, is it?
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2125#discussion_r56035879
>    2. https://github.com/sulu-io/sulu/pull/2125/files#r56035879
>    3. https://github.com/sulu-io/sulu/pull/2125/files#r56035879